app/assets/images/index.js

```diff
@@ -25,6 +25,8 @@ export { default as miner } from './mining.gif';
 export { default as welcomeBack } from './welcome_back@2x.png';
 export { default as copyIcon } from './copy_green@2x.png';
 export { default as shieldIconGreen } from './security_logo_green@2x.png';
+export { default as shieldIconGreenOne } from './security_logo_green_one@2x.png';
```

Comment:
there is other shieldIcon? If not - rename to shieldIconGreen

Reply:
Yes. the regular shield icon is not numbered. this one has "1", the other one has a "!" in its center.